app/routes/users/show.js

```diff
@@ -1,37 +1,43 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 import { set } from '@ember/object';
-// import RSVP from 'rsvp';
-// import fetch from 'fetch';
-// import { request, GraphQLClient } from 'graphql-request';
-
-// const query = `{
-//   person(id: "https://orcid.org/0000-0002-8862-1404") {
-//     givenName
-//     familyName
-//     orcid: id
-//     citationCount
-//     viewCount
-//     downloadCount
-//   }
-// }`;
 
+import { all } from 'rsvp';
+import { assign } from '@ember/polyfills';
 
 export default Route.extend({
   can: service(),
   features: service(),
   headData: service(),
 
   model(params) {
+
     let self = this;
-    return this.store.findRecord('user', params.user_id).then(function(user) {
-      set(self, 'headData.title', user.name);
-      return user;
-    }).catch(function(reason) {
-      console.debug(reason);
-
-      self.get('flashMessages').warning('Fabrica is currently unavailable due to a DataCite API problem. We apologize for the inconvenience and are working hard to restore the service. Please check back later or contact DataCite Support if you have a question.');
-      self.transitionTo('/');
+
+    let parameters = {
+      'user-id': `${params.user_id}`,
+      'mix-in': 'metrics',
+    };
+
+    // eslint-disable-next-line no-undef
+    return new Promise(function(resolve, reject) {
+      all([
+        self.store.findRecord('user', params.user_id).then(function(user) {
+          set(self, 'headData.title', user.name);
+          return user;
+        }).catch(function(reason) {
+          console.debug(reason);
+
+          self.get('flashMessages').warning('Fabrica is currently unavailable due to a DataCite API problem. We apologize for the inconvenience and are working hard to restore the service. Please check back later or contact DataCite Support if you have a question.');
+          self.transitionTo('/');
+        }),
+        self.store.query('doi', parameters).then(function(result) {
+          return result.meta;
+        }).catch(error => console.log(error)) ])
+        .then(function([ hashA, hashB ]) {
```

Comment:
can you explain this line?

Reply:
It's just a merge of the user data and the metrics data. We merge them to use them in the model hook together